src/util/FactorUtil.js

```diff
@@ -141,20 +141,11 @@ define(['okta' , './CookieUtil'], function (Okta, CookieUtil) {
   };
 
   fn.getRememberDeviceValue = function (settings, appState) {
-    var username = appState && appState.get('username');
     var rememberDeviceAlways = settings && settings.get('features.rememberDeviceAlways');
-    var rememberDeviceUsername = CookieUtil.getCookieDeviceUsername();
-    var rememberDevice = false;
-
-    // rememberDevice is true if 'rememberDeviceAlways' is on or
-    // if the last username is same as the current username.
-    if (rememberDeviceAlways) {
-      rememberDevice = true;
-    } else if (rememberDeviceUsername || username) {
-      rememberDevice = (rememberDeviceUsername === username);
-    }
-
-    return rememberDevice;
+    var rememberDeviceValueFromServer = appState && appState.get('rememberDeviceByDefault');
+    // rememberDevice is true if 'rememberDeviceAlways' is on (feature flag)
+    // or if the server returns true for this user/device
+    return rememberDeviceAlways || rememberDeviceValueFromServer;
```

Comment:
We will not rememberDeviceAlways config too, since we get this information from the server.